Pokedex_ios.xcodeproj/xcshareddata/xcschemes/Pokedex_iosUITests.xcscheme

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
these files changed are the problem, other branches work great, but I don't know why you have these changes in xcscheme

Reply:
I merged the develop branch into mine.

Reply:
I replaced all files in the branch from develop, then added the height filtering over it. Hope it works!

Reply:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/20176876/179647242-fcfec960-e98b-4289-9a1d-c39624671070.png)

Circleci running successfully!